packages/@webex/plugin-meetings/test/unit/spec/meeting/request.js

```diff
@@ -195,7 +197,42 @@ describe('plugin-meetings', () => {
         assert.equal(requestParams.body.device.countryCode, 'US');
         assert.equal(requestParams.body.permissionToken, 'permission-token');
         assert.equal(requestParams.body.device.regionCode, 'WEST-COAST');
-      });
+        assert.include(requestParams.body.device.localIp, '127.0.0');
```

Comment:
question: Why is this an `include` instead of `equal`? The only way we know that the IP address is being anonymized is with this check, this presumably should check for `'127.0.0.0'`? Really I'd like to spy on the anonymize function to ensure it is being used correctly, but this simple check would be better than nothing.

Reply:
Well if it's getting anonymized, the last 4 digits would always change. I think what would have to change here is that I will have to stub the anonymize function and have it always return the same result, and then use an equal. That's fine I can do that and it's definitely a "better" test

Reply:
I have changed this, please have a look and let me know if you're not satisfied with it. I didn't feel like it was the best idea to retest the anonymize function, felt the test should be done in the call-diagnostic metrics util source itself, not here as well, so i felt like includes was good enough. In any case, i've changed it now using anonymize as a stub.